Kentico.Kontent.Management/Modules/ResiliencePolicy/DefaultResiliencePolicyProvider.cs

```diff
@@ -10,6 +14,22 @@ namespace Kentico.Kontent.Management.Modules.ResiliencePolicy;
 /// </summary>
 public class DefaultResiliencePolicyProvider : IResiliencePolicyProvider
 {
+    private static readonly HttpStatusCode[] StatusCodesToRetry =
+    {
+            HttpStatusCode.RequestTimeout,
+            (HttpStatusCode)429, // Too Many Requests
+            HttpStatusCode.InternalServerError,
+            HttpStatusCode.BadGateway,
+            HttpStatusCode.ServiceUnavailable,
+            HttpStatusCode.GatewayTimeout,
+    };
+    
+    private static readonly HttpStatusCode[] StatusCodesWithPossibleRetryHeader =
+    {
+            (HttpStatusCode)429, // Too Many Requests
+            HttpStatusCode.ServiceUnavailable,
```

Comment:
(I know it is also in DAPI SDK)
Is it valid that this error (503) can also return retry-after? - It does not seems like ti in https://kontent.ai/learn/reference/management-api-v2/#section/Errors/HTTP-status-codes-summary It

Reply:
I think it's unusual to return a `retry-after` header for any response other than 429. 5xx tells me it's an internal server error, so I wouldn't expect any extra information.
However, [MDN page on the `retry-after` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After) states that this usage "indicates how long the service is expected to be unavailable." Is this what we're signaling with the header? If so, I guess it's okay. If not, I wouldn't include it.